scuttlebutt/src/failure_detector.rs

```diff
@@ -72,7 +72,7 @@ impl FailureDetector {
             debug!(node_id = ?node_id, phi = phi, "updating node liveliness");
             if phi > self.config.phi_threshold {
                 self.live_nodes.remove(node_id);
```

Comment:
super ultra nitpick.

I like the idea to emphasive on the fact taht we move the node from live to dead.

for instance via:

```
if let Some(node_id) = self.live_nodes.remove(node_id) {
    self.dead_nodes.insert(node_id, Instant::now());
}
```

Reply:
( no need to fix that'ss probably just me)

Reply:
This might not be a nitpick. it really makes the code speaks for itself TBH. which is always great to have